client/src/extension.ts

```diff
@@ -89,8 +90,26 @@ export function activate(context: ExtensionContext) {
         ExecYseopCliCommand(yseopCliPath, 'package');
     });
 
-    const infoCmd = commands.registerCommand(`${yseopmlSectionName}.info`, () => {
-        ExecYseopCliCommand(yseopCliPath, 'info');
+    const infoCmd = commands.registerCommand(`${yseopmlSectionName}.config`, () => {
+        ExecYseopCliCommand(yseopCliPath, 'config');
+    });
+
+    const libsInstallCmd = commands.registerCommand(`${yseopmlSectionName}.libs`, () => {
+        ExecYseopCliCommand(yseopCliPath, 'libs', 'install', '-R');
+    });
+
+    const deployCmd = commands.registerCommand(`${yseopmlSectionName}.managerDeploy`, (ymaUri: Uri) => {
+        window
+            .showInputBox({ prompt: 'Leave this box empty to use the default env.', placeHolder: 'DEMO' })
+            .then((env) => {
+                if (!!env && env.length > 0) {
+                    window.showInformationMessage(`Deploying YMA ${ymaUri.fsPath} on env “${env}”.`);
```

Comment:
* “on”? 🤔 why not “with”, like in the case below?
* as before, “environment” seems friendlier than “env”. Even the option’s long name is “environment”; let’s try to align a bit and avoid unnecessary abbreviations.

Reply:
OK!